app/components/companion_windows_component.rb

```diff
@@ -63,6 +63,7 @@ def outer_container_data
         data[:fullscreen_target] = 'area'
         data[:fullscreen_close_value] = 'Exit full screen'
       end
+      data[:companion_window_expanded_value] = render_content_list_panel?
```

Comment:
This data attribute name makes it sound like it expresses the current state of the companion window. Maybe something like `companion_window_auto_expand` would be clearer?

Reply:
That makes sense. Updated.